bundles/org.openhab.core.model.rule/src/org/openhab/core/model/rule/Rules.xtext

```diff
@@ -79,6 +80,10 @@ DateTimeTrigger:
     'Time' 'is' item=ItemName
```

Comment:
```suggestion
    'Time' 'is' item=ItemName (timeOnly?='timeOnly')?
```

Wouldn't this work? It should create a boolean `timeOnly`.

Reply:
Changed it now to use this approach